cmd/bucket-handlers.go

```diff
@@ -376,6 +376,13 @@ func (api objectAPIHandlers) DeleteMultipleObjectsHandler(w http.ResponseWriter,
 		return
 	}
 
+	// Before proceeding validate if bucket exists.
+	_, err := objectAPI.GetBucketInfo(ctx, bucket)
```

Comment:
Why is this needed ? also could you add a visible comment to say this is called even if the request is not authenticated yet

Reply:
That is just for S3 compatibility @vadmeste

Reply:
> That is just for S3 compatibility @vadmeste

yes. Better to have a visible comment, because this handler has really different auth style, LGTM otherwise

Reply:
Yes, the authentication should be handled in a different PR

Reply:
> Yes, the authentication should be handled in a different PR

oh okay, I thought we are not doing that.